### PR TITLE
Add medical-info-page design layout

### PR DESCRIPTION
**Pull request by Gtstride**

### CODE REVIEW

File: client/src/pages/Medical.js [hunk 1]
```diff
@@ -1,17 +1,109 @@
-import axios from "axios";
+// import { getAirtableRecord } from "../utils/airtable";
+// import { getLocalStorageJson } from "../utils/local-storage";
+
+// const CONTAINER_STYLES = {
```
Comment (robinv85) [inline client/src/pages/Medical.js:4]: Maybe remove all unused code? Why keep these commented out?

File: client/src/pages/Medical.js [hunk 2]
```diff
@@ -20,13 +112,38 @@ const INITIAL_STATE = {
...
+
 const needHelpAnswers = getLocalStorageJson("needHelpAnswers") || [];
 console.log({ needHelpAnswers });
 
 const getGeoLocation = () => needHelpAnswers?.location;
 
-export const Medical = () => {
+export const Medical = (props) => {
   const [state, setState] = useState(INITIAL_STATE);
+  console.log("render medical page", { props });
```
Comment (robinv85) [inline client/src/pages/Medical.js:146]: Forgot console.log?

File: client/src/pages/Medical.js [hunk 3]
```diff
@@ -64,30 +181,48 @@ export const Medical = () => {
...
+      <StyledWelcome level={3} size="xlarge">
+        Please Call Your Local Emergency Number
+      </StyledWelcome>
+      <OnboardingContainer style={{ display: "flex", flexWrap: "wrap" }}>
+        <FlexChild>
+          <ImageButton
+            type="ghost"
+            inactiveImg={nearestHospitalUnselected}
+            activeImg={nearestHospitalSelected}
+            onClick={() => props.history.push("/nearest-hospital")}
```
Comment (robinv85) [inline client/src/pages/Medical.js:200]: Maybe this is out of scope of this pull request
but should these be like "tabs" inside the "medical" page or change the full "page" ?

File: client/src/pages/Medical.js (hunk 1 above)
Comment (Gtstride) [reply client/src/pages/Medical.js:4]: Thought, we might still want to come back to them later-on
